std/typecons.d

```diff
@@ -3384,6 +3389,21 @@ auto nullable(T)(T t)
     assert(destructorCalled);
 }
 
+// check that toHash on Nullable is forwarded to the contained type
+@safe unittest
+{
+    struct S
+    {
+        size_t toHash() const @safe pure nothrow { return 5; }
+    }
+
+    Nullable!S s1 = S();
+    Nullable!S s2 = Nullable!S();
+
+    assert(typeid(Nullable!S).getHash(&s1) == 5);
+    assert(typeid(Nullable!S).getHash(&s2) == 0);
+}
```

Comment:
Taking the address of locals in a `@safe` unittest? https://run.dlang.io/is/6VLvzX
How did this pass the CI tests?!

Reply:
Pinging @MartinNowak; this is a blocker for the final (obviously).

Reply:
Were the tests run with DIP1000? It works with that.

Reply:
Omg... https://github.com/dlang/phobos/blob/master/dip1000.mak - unused by LDC, we use CMake. This is the first module failing to compile without it.